config/owasp/suppressions.xml

```diff
@@ -7,6 +7,8 @@
         <filePath regex="true">.*spring-security-(config|crypto|web|test|core)-5\.2\.[0-2]\.RELEASE\.jar</filePath>
         <cpe>cpe:/a:pivotal_software:spring_security</cpe>
         <cve>CVE-2018-1258</cve>
+        <cve>CVE-2020-5407</cve>
```

Comment:
Upgrading your spring boot should allow you to remove these suppressing FYI

Reply:
Thanks Jack, spring boot upgrade is being handled in a seperate PR.